### ISSUE TITLE
Problem dealing with alerts on 3.0.10

### ISSUE THREAD

**Issue by berangerqtn** [simcen/alert_manager#306]
When receiving an alert, alert_manager.py tries to access fields in the search, even if this field does not exist.

```python
 if ('impact' in result or result['impact'] != ''):
        settings['impact'] = result['impact']
    elif ('impact' not in cfg or cfg['impact'] == ''):
        settings['impact'] = ''
    else:
        settings['impact'] = cfg['impact']

    if ('urgency' in result or result['urgency'] != ''):
        settings['urgency'] = result['urgency']
    elif ('urgency' not in cfg or cfg['urgency'] == ''):
        settings['urgency'] = ''
    else:
        settings['urgency'] = cfg['urgency']
```

Should be an and condition not a or.

Posting PR asap

**Comment by my2ndhead** [comment]
Can you please create a pull request for develop? Just merged my latest fixes

**Comment by berangerqtn** [comment]
Yes, I open the PR right now

**Comment by berangerqtn** [comment]
Solved on PR #309
